src/test/java/htsjdk/samtools/ValidateSamFileTest.java

```diff
@@ -89,7 +89,7 @@ public void testValidCRAMFileWithoutSeqDict() throws Exception {
     public void testSamFileVersion1pt5() throws Exception {
         final SamReader samReader = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.SILENT).open(new File(TEST_DATA_DIR, "test_samfile_version_1pt5.bam"));
         final Histogram<String> results = executeValidation(samReader, null, IndexValidationStringency.EXHAUSTIVE);
-        Assert.assertTrue(results.isEmpty());
+        Assert.assertFalse(results.isEmpty());
```

Comment:
Instead of `assertFalse()` this should assert the number of results and the values for each expected result, if possible.

Reply:
Assert is changed to check results count.